release-notes/1.19.0.md

```diff
@@ -0,0 +1,69 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud < release-version >.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* The AMS API documentation created using OpenAPI specification is available on the AMS host at `/api/1.0/swagger.json`.
+* Pagination and filtering support is enabled for containers and applications in the AMS API. This has made several dashboard enhancements possible. You can also use it if you want to interact directly with the AMS using API calls.
+* AV1 hardware accelerated video encoding on NVIDIA GPUs is enabled.
+
```

Comment:
```suggestion
* cgroup v1 emulation layer for cgroup v2-only hosts

```
Do you think we need to explain more about this? We could also quickly explain that it's required by Android for tracking and assigning resources to different processes.

Reply:
That sounds good enough!